src/select.coffee

```diff
@@ -80,8 +81,19 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
   # we need to set tabindex so that div responds to key events
   tabindex: 0
 
+  ###*
+  * Flag controlling the dropdown's appearance of the select component
+  * @type {Boolean}
+  ###
   showDropdown: no
 
+  ###*
+  * Flag indicating that dropdown will always appear as part of the select
+  * component
+  * @type {Boolean}
+  ###
+  lockDropdownOnShow: no
```

Comment:
It's not `init with Dropdown` but instead, it will never collapse the dropdown. 
So Jordan+Kevin's suggestions will become `shouldLockDropdownOnShow`? And yeah, we have 80 characters length limit.